features/conveyancer_adds_title_number_to_deed.feature

```diff
@@ -9,14 +9,18 @@ Acceptance Criteria
   Response code returned for successful and unsuccessful request
   A link to the resource must be returned as part of the response
 
+@us6
 Scenario: Add Multiple Title Numbers
-  Given I have deed data with two title numbers
+  Given I setup a deed with <1> borrowers
+  And I amend "title_number" to "DT567568, DN573253" for borrower <1>
```

Comment:
Yeh sorry as discussed the title number is not something that is really associated to a borrower so wonder if this might be a bit confusing, especially when changing something that might have similar names for borrower and deed like 'token' or something like that further down the line. Don't know if theres a way to make the borrower bit optional or have two seperate steps one for overwriting borrower data and one for the deed data.